phpstan.neon

```diff
@@ -13,7 +13,4 @@ parameters:
     doctrine:
         objectManagerLoader: tests/object-manager.php
 
-    bootstrapFiles:
-        - vendor/bin/.phpunit/phpunit-8.5-0/vendor/autoload.php
```

Comment:
I always get PHPUnit 9.6 locally, so to run PHPStan I had to edit this committed files...